packages/@dcl/sdk/cli/commands/preview/wire.ts

```diff
@@ -1,13 +1,25 @@
 import path from 'path'
 import fs from 'fs-extra'
+import chokidar from 'chokidar'
+import { sdk } from '@dcl/schemas'
+import { WebSocket } from 'ws'
+import { resolve } from 'path'
 
+import { getSceneFile } from '../preview/project'
+import { CliComponents } from '../../components'
 import { setupBffAndComms } from './bff'
 import { Router } from '@well-known-components/http-server'
 import { setupEcs6Endpoints } from './endpoints'
 import { CliError } from '../../utils/error'
 import { PreviewComponents } from './types'
+import { upgradeWebSocketResponse } from '@well-known-components/http-server/dist/ws'
 
-export async function wire(dir: string, components: PreviewComponents) {
+export async function wire(
+  components: Pick<CliComponents, 'fs'>,
+  dir: string,
+  previewComponents: PreviewComponents,
```

Comment:
```suggestion
  components: CliComponents,
  dir: string,
```

Please unify components into CliComponents, delete PreviewComponents